.travis.yml

```diff
@@ -48,6 +48,9 @@ script:
 - |
   # Test cargo integration
   (cd cargo-miri-test && ./run-test.py)
+- |
+  # make sure benchmarks build
+  cargo build --all-targets
```

Comment:
Could you add `--all-targets` to the `cargo build` above, so that miri doesn't get build twice (once in release mode and once in debug mode)

Reply:
Also please adjust the appveyor config the same way as well.

Reply:
Done